cf-custom-resources/lib/copy-assets.js

```diff
@@ -0,0 +1,21 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+"use strict";
+
+const aws = require("aws-sdk");
+
+/**
+ * Main handler, invoked by Lambda
+ */
+exports.handler = async function (event, context, callback) {
+	const s3 = new aws.S3();
```

Comment:
nit: consider moving the initialization of the client outside of the handler (https://docs.aws.amazon.com/lambda/latest/operatorguide/static-initialization.html)

I think it might make a performance difference if the SFN state machine is invoking this Lambda fn for each object. 💭 

It would have been cool if we could have used the aws-sdk-v3 and just imported the s3 client :cool-crying:

Reply:
i actually had the initialization outside initially, but https://www.npmjs.com/package/aws-sdk-mock 😔 

> NB: The AWS Service needs to be initialised inside the function being tested in order for the SDK method to be mocked